tests/components/blebox/test_sensor.py

```diff
@@ -0,0 +1,88 @@
+"""Blebox sensors tests."""
+
+import logging
+
+import blebox_uniapi
+import pytest
+
+from homeassistant.const import (
+    ATTR_DEVICE_CLASS,
+    ATTR_UNIT_OF_MEASUREMENT,
+    DEVICE_CLASS_TEMPERATURE,
+    STATE_UNKNOWN,
+    TEMP_CELSIUS,
+)
+
+from .conftest import async_setup_entity, mock_feature
+
+from tests.async_mock import AsyncMock, PropertyMock
+
+
+@pytest.fixture(name="tempsensor")
+def tempsensor_fixture():
+    """Return a default sensor mock."""
+    feature = mock_feature(
+        "sensors",
+        blebox_uniapi.sensor.Temperature,
+        unique_id="BleBox-tempSensor-1afe34db9437-0.temperature",
+        full_name="tempSensor-0.temperature",
+        device_class="temperature",
+        unit="celsius",
+        current=None,
+    )
+    product = feature.product
+    type(product).name = PropertyMock(return_value="My temperature sensor")
+    type(product).model = PropertyMock(return_value="tempSensor")
+    return (feature, "sensor.tempsensor_0_temperature")
+
+
+async def test_init(tempsensor, hass, config):
+    """Test sensor default state."""
+
+    _, entity_id = tempsensor
+    entry = await async_setup_entity(hass, config, entity_id)
+    assert entry.unique_id == "BleBox-tempSensor-1afe34db9437-0.temperature"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)